openedx/core/lib/xblock_builtin/xblock_discussion/xblock_discussion.py

```diff
@@ -108,6 +111,32 @@ def student_view(self, context=None):
         """
         fragment = Fragment()
 
+        # Head dependencies
+
+        discussion_vendor_js = ['js/jquery.autocomplete.js', 'js/src/tooltip_manager.js']
+
+        discussion_vendor_js += settings.PIPELINE_JS['discussion_vendor']['source_filenames']
```

Comment:
@itsjeyd I would expect it to just work, but have you checked if it still works in "pipeline" mode? I can't recall how exactly you switch to that - the idea is that pipeline is disabled in development env (default env for devstack) and all the files are just added one-by-one. In production, pipeline is enabled, so all those files are concatenated-minified-etc. so only one file per `PIPELINE_JS['entry']` is delivered. It would be nice to be sure that it still delivers one file when pipeline is enabled.